certsubmitter/certsubmitter.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package certsubmitter periodically issues a certificate or pre-certificate
+// and submits it to a CT Log.
+package certsubmitter
+
+import (
+	"context"
+	"time"
+
+	"github.com/golang/glog"
+	ct "github.com/google/certificate-transparency-go"
+	"github.com/google/certificate-transparency-go/schedule"
+	"github.com/google/monologue/apicall"
+	"github.com/google/monologue/certgen"
+	"github.com/google/monologue/client"
+	"github.com/google/monologue/ctlog"
+	"github.com/google/monologue/storage"
+)
+
+var logStr = "Certificate Submitter"
```

Comment:
Make this `const` instead? Also unclear to me why it's not just inlined in the log messages themselves, perhaps in a shortened way ("certsubmitter", maybe?)...

Reply:
Every module has this logStr variable with whatever their module is.  I've changed this and the others to be consts too.

By having it defined up at the top it can be easily used throughout the code in all of the log messages without accidental misspelling etc.  And I don't think there's any need for shortening - for log messages having something clear and readable is my goal.

Reply:
SGTM